nvflare/edge/emulator/run_emulator.py

```diff
@@ -59,14 +64,8 @@ def run_emulator(endpoint_url: str, num: int):
     )
 
     n = len(sys.argv)
-    if n >= 2:
-        endpoint = sys.argv[1]
-    else:
-        endpoint = "http://localhost:9007"
-
-    if n >= 3:
-        num_devices = int(sys.argv[2])
-    else:
-        num_devices = 4
+    if n != 2:
+        print("Usage: python run_emulator.py <config_file>")
+        sys.exit(1)
 
-    run_emulator(endpoint, num_devices)
+    run_emulator(sys.argv[1])
```

Comment:
I wonder why you don't user Python's argparse?

Reply:
Changed to user argparse.